packages/odyssey-ie-11-sandbox/src/App.tsx

```diff
@@ -0,0 +1,518 @@
+          withContainer
+        >
+          <Table.Header>
+            <Table.Row>
+              <Table.HeaderCell scope="col">
+                <Table.SortButton
+                  direction="asc"
+                  unsortedIconTitle="Unsorted"
+                  ascendingIconTitle="Ascending"
+                  descendingIconTitle="Descending"
+                  screenReaderCallToAction="click to sort"
+                >
+                  Planet
+                </Table.SortButton>
+              </Table.HeaderCell>
+              <Table.HeaderCell scope="col" format={"num"}>
+                Radius (km)
+              </Table.HeaderCell>
+              <Table.HeaderCell scope="col">Type</Table.HeaderCell>
+              <Table.HeaderCell scope="col" format={"date"}>
+                Perihelion date
+              </Table.HeaderCell>
+            </Table.Row>
+          </Table.Header>
+          <Table.Body>
+            <Table.Row>
+              <Table.DataCell>Jupiter</Table.DataCell>
+              <Table.DataCell format={"num"}>69,911</Table.DataCell>
+              <Table.DataCell>Gas giant</Table.DataCell>
+              <Table.DataCell format={"date"}>January 21, 2023</Table.DataCell>
+            </Table.Row>
+            <Table.Row>
+              <Table.DataCell>Earth</Table.DataCell>
+              <Table.DataCell format={"num"}>6,371</Table.DataCell>
+              <Table.DataCell>Terrestrial</Table.DataCell>
+              <Table.DataCell format={"date"}>January 2, 2021</Table.DataCell>
+            </Table.Row>
+            <Table.Row>
+              <Table.DataCell>Mercury</Table.DataCell>
+              <Table.DataCell format={"num"}>1,737</Table.DataCell>
+              <Table.DataCell>Terrestrial</Table.DataCell>
+              <Table.DataCell format={"date"}>&ndash;</Table.DataCell>
+            </Table.Row>
+          </Table.Body>
+        </Table>
+      </ErrorBoundary>
+
+      <Status
+        descriptor="Tag List"
+        label="Status Label"
+        labelHidden
+        variant="neutral"
+      />
+      <ErrorBoundary>
+        <TagList tags={["Item one", "Item two", "Item three"]} />
+      </ErrorBoundary>
+
+      <Status
+        descriptor="Text"
+        label="Status Label"
+        labelHidden
+        variant="neutral"
+      />
+      <ErrorBoundary>
+        <Text>Text</Text>
+      </ErrorBoundary>
+
+      <Status
+        descriptor="TextArea"
+        label="Status Label"
+        labelHidden
+        variant="caution"
+      />
+      <ErrorBoundary>
+        <TextArea hint="Descriptive field hint" label="TextArea" />
+      </ErrorBoundary>
+
+      <Status
+        descriptor="TextInput"
+        label="Status Label"
+        labelHidden
+        variant="caution"
+      />
+      <ErrorBoundary>
+        <TextInput label="TextInput" />
+      </ErrorBoundary>
+
+      <Status
+        descriptor="Toast"
+        label="Status Label"
+        labelHidden
+        variant="danger"
```

Comment:
what are we indicating with the status indicators here? problematic components that you've identified in ie11 = red? what about grey or yellow?

Reply:
Yellow = has visual differences but is working
Gray = Nothing seems to be wrong but the full API has not been tested